### PR TITLE
stubs and test for rnaseq modules - part II

### PR DESCRIPTION
**Pull request by maxulysse**
## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Comment by muffato** [comment]
@maxulysse : `umitools/extract` didn't pass the tests. Can you take a look ?

**Comment by maxulysse** [comment]
> @maxulysse : `umitools/extract` didn't pass the tests (nf-test and linting). Can you take a look ?

Nope, sorry, I'm out celebrating midsummer.
I'll have a look next week if I find time. 
But if anyone wants to take over, do not hesitate to push on my branch

**Comment by SPPearce** [comment]
> @maxulysse : `umitools/extract` didn't pass the tests (nf-test and linting). Can you take a look ?

It has 115k lines in the snap, something not quite right in that ;)

**Comment by maxulysse** [comment]
I'll see what's happening at the airport

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please :pray:

File: modules/nf-core/ucsc/bedclip/tests/main.nf.test [hunk 1]
```diff
@@ -30,4 +30,29 @@ nextflow_process {
             )
         }
     }
+
+    test("sarscov2 - stub") {
+        config "./nextflow.config"
+
```
Comment (SPPearce) [inline modules/nf-core/ucsc/bedclip/tests/main.nf.test:36]:
```suggestion
```

**Review by SPPearce — APPROVED** [review]

File: modules/nf-core/rsem/preparereference/main.nf [hunk 2]
```diff
@@ -69,4 +69,15 @@ process RSEM_PREPAREREFERENCE {
         END_VERSIONS
         """
     }
+
+    stub:
+    """
+    touch genome.transcripts.fa
```
Comment (JoseEspinosa) [inline modules/nf-core/rsem/preparereference/main.nf:75]: It should be a silly reason, but why is not complaining about not finding the `index` when running the stub?

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Just have a question 🤔

File: modules/nf-core/rsem/preparereference/main.nf (hunk 2 above)
Comment (JoseEspinosa) [reply modules/nf-core/rsem/preparereference/main.nf:75]: Ah true, it was a silly reason, should have looked into the `input` 😄
Thanks!

**Pull request merged**